diff --git a/dist/index.mjs b/dist/index.mjs
index beadb1174a84b41e3ebb1c805bf431f8c2b23719..bb67875ef8621680f967f288aa04fa7e36aaf107 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3230,7 +3230,7 @@ const pluginResources = ({ contentScripts = {} }) => {
               ...rest
             })).filter(({ resources }) => resources.length);
             manifest.web_accessible_resources.push({
-              use_dynamic_url: true,
+              use_dynamic_url: false,
               matches: ["<all_urls>"],
               resources: ["**/*", "*"]
             });
@@ -3312,7 +3312,7 @@ const pluginResources = ({ contentScripts = {} }) => {
                     const resource = {
                       matches: script.matches,
                       resources: [...assets, ...imports],
-                      use_dynamic_url: true
+                      use_dynamic_url: false
                     };
                     if (css.size)
                       if (injectCss) {
@@ -3513,7 +3513,7 @@ const pluginBackground = () => {
 const defineManifest = (manifest) => manifest;
 const defineDynamicResource = ({
   matches = ["http://*/*", "https://*/*"],
-  use_dynamic_url = true
+  use_dynamic_url = false
 }) => ({
   matches,
   resources: [dynamicResourcesName],
